wiki/en/Installation-for-Linux.md

```diff
@@ -54,24 +54,43 @@ For installers on other distributions, see their package managers and [Repology]
 
 ## Set up your hardware
 
+The Jamulus Client needs to connect to a running [JACK](https://jackaudio.org/) server in order to start. You need to find out which is the low-latency sound server for your distribution.
+- If your distribution uses [JACK](https://jackaudio.org/), see how to [configure JACK with QjackCtl](Installation-for-Linux#configure-jack-with-qjackctl).
+- If your distribution uses PipeWire, see how to [configure PipeWire](Installation-for-Linux#configure-pipewire).
+
 ### Configure JACK with QjackCtl
 
-Jamulus Clients need [JACK](https://jackaudio.org/) to run, but you need to configure that first. The recommended method is to use `QjackCtl`.
+To run a [JACK](https://jackaudio.org/) server, the recommended method is to use `QjackCtl`.
+
+1. Launch QjackCtl. You will see the **Qt JACK Control utility main page**.
+2. Configure your sound hardware as follows (except where noted):
```

Comment:
Except where noted -> what does that mean? I'd rather get rid of that.

Reply:
@pljones certainly knows.
It is in this commit:
commit 87aff02ce2069de4a9d4c8d7c5e3750c3018e36f

Reply:
Did you mean some other commit?
```
87aff02ce wiki/en/Installation-for-Linux.md    (Julien Taverna     2023-12-30 22:27:39 +0100  65) 2. Configure your audio hardware as follows (except where noted):
```

Reply:
Ah, you mean here: https://github.com/jamulussoftware/jamuluswebsite/pull/980#discussion_r1438545180

I think I was expecting to see some explanation of how "the exact settings for JACK will depend on what your audio hardware is capable of" would affect the settings chosen, those being noted.

Reply:
```suggestion
2. Configure your sound hardware as follows:
```

Or 

```suggestion
2. Configure your sound hardware (your configuration should look comparable, but may slightly differ for each audio device):
```